src/Login/index.test.js

```diff
@@ -8,68 +8,121 @@ import Login from './index';
 import QUERY_USERS from '../graphql/queries';
 import { HOME_PATH, LOGIN_PATH } from '../paths';
 
-const mockedUsersResponse = {
-  request: { query: QUERY_USERS },
-  result: {
-    data: {
-      users: [
-        {
-          username: 'User',
-          password: 'password',
-          id: 1,
-        },
-      ],
+const mocks = [
+  {
+    request: { query: QUERY_USERS },
+    result: {
+      data: {
+        users: [
```

Comment:
@Aaron-Goldman Would there be an issue with having multiple users in here rather than just one? My main goal is to mimic a real situation where we test that this code would be solid when we are searching for a user that's part of a large collections of existing users: 
`result.data.users.find((u) => u.username === username && u.password === password);`

Ideally we want to mock having at least 3 users in the database like this:
```
      users: [
        {
          username: 'Danny',
          password: '632569',
          id: 1,
        },
        {
          username: 'James',
          password: '12345',
          id: 2,
        },
        {
          username: 'William',
          password: '678912',
          id: 3,
        }
      ]
```

But if we only have one user in the mock, our test won't be as efficient I think, does it make sense?